digits/standard-networks/tensorflow/alexnet_slim.py

```diff
@@ -0,0 +1,38 @@
+from model import Tower
```

Comment:
yeah equivalent. one could even use keras. Tensorflow vanilla is really not concise, which is why keras is so popular :D